2023-03-17 16:23:44,114 INFO    StreamThr :26728 [internal.py:wandb_internal():87] W&B internal server running at pid: 26728, started at: 2023-03-17 16:23:44.114290
2023-03-17 16:23:44,114 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status
2023-03-17 16:23:44,124 INFO    WriterThread:26728 [datastore.py:open_for_write():85] open: d:\CS6901\wandb\run-20230317_162344-dytlg310\run-dytlg310.wandb
2023-03-17 16:23:44,124 DEBUG   SenderThread:26728 [sender.py:send():336] send: header
2023-03-17 16:23:44,184 DEBUG   SenderThread:26728 [sender.py:send():336] send: run
2023-03-17 16:23:45,892 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: check_version
2023-03-17 16:23:45,893 INFO    SenderThread:26728 [dir_watcher.py:__init__():219] watching files in: d:\CS6901\wandb\run-20230317_162344-dytlg310\files
2023-03-17 16:23:45,894 INFO    SenderThread:26728 [sender.py:_start_run_threads():1078] run started: dytlg310 with start time 1679050424.11429
2023-03-17 16:23:45,894 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: summary_record
2023-03-17 16:23:45,895 INFO    SenderThread:26728 [sender.py:_save_file():1332] saving file wandb-summary.json with policy end
2023-03-17 16:23:45,895 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: check_version
2023-03-17 16:23:45,959 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: run_start
2023-03-17 16:23:46,195 DEBUG   HandlerThread:26728 [system_info.py:__init__():31] System info init
2023-03-17 16:23:46,196 DEBUG   HandlerThread:26728 [system_info.py:__init__():46] System info init done
2023-03-17 16:23:46,196 INFO    HandlerThread:26728 [system_monitor.py:start():183] Starting system monitor
2023-03-17 16:23:46,204 INFO    SystemMonitor:26728 [system_monitor.py:_start():147] Starting system asset monitoring threads
2023-03-17 16:23:46,205 INFO    HandlerThread:26728 [system_monitor.py:probe():204] Collecting system info
2023-03-17 16:23:46,213 INFO    SystemMonitor:26728 [interfaces.py:start():187] Started cpu monitoring
2023-03-17 16:23:46,214 INFO    SystemMonitor:26728 [interfaces.py:start():187] Started disk monitoring
2023-03-17 16:23:46,217 INFO    SystemMonitor:26728 [interfaces.py:start():187] Started gpu monitoring
2023-03-17 16:23:46,219 INFO    SystemMonitor:26728 [interfaces.py:start():187] Started memory monitoring
2023-03-17 16:23:46,256 INFO    SystemMonitor:26728 [interfaces.py:start():187] Started network monitoring
2023-03-17 16:23:46,265 DEBUG   HandlerThread:26728 [system_info.py:probe():195] Probing system
2023-03-17 16:23:46,267 DEBUG   HandlerThread:26728 [system_info.py:_probe_git():180] Probing git
2023-03-17 16:23:46,267 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:46,268 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:46,355 DEBUG   HandlerThread:26728 [system_info.py:_probe_git():188] Probing git done
2023-03-17 16:23:46,355 DEBUG   HandlerThread:26728 [system_info.py:probe():240] Probing system done
2023-03-17 16:23:46,355 DEBUG   HandlerThread:26728 [system_monitor.py:probe():213] {'os': 'Windows-10-10.0.22621-SP0', 'python': '3.9.7', 'heartbeatAt': '2023-03-17T10:53:46.266849', 'startedAt': '2023-03-17T10:53:44.094277', 'docker': None, 'cuda': None, 'args': None, 'state': 'running', 'program': '<python with no main file>', 'git': {'remote': 'https://github.com/Vishwanath1999/CS6901.git', 'commit': '9016d3702b93ee7a00952ac71741d417ea21dcff'}, 'email': 'viswasankar98@gmail.com', 'root': 'D:/CS6901', 'host': 'darkAngel', 'username': 'HP', 'executable': 'c:\\Users\\HP\\AppData\\Local\\Programs\\Python\\Python39\\python.exe', 'cpu_count': 4, 'cpu_count_logical': 8, 'cpu_freq': {'current': 1600.0, 'min': 0.0, 'max': 1800.0}, 'cpu_freq_per_core': [{'current': 1600.0, 'min': 0.0, 'max': 1800.0}], 'disk': {'total': 367.49511337280273, 'used': 135.9652328491211}, 'gpu': 'NVIDIA GeForce 940MX', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce 940MX', 'memory_total': 2147483648}], 'memory': {'total': 15.88699722290039}}
2023-03-17 16:23:46,355 INFO    HandlerThread:26728 [system_monitor.py:probe():214] Finished collecting system info
2023-03-17 16:23:46,355 INFO    HandlerThread:26728 [system_monitor.py:probe():217] Publishing system info
2023-03-17 16:23:46,355 DEBUG   HandlerThread:26728 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-03-17 16:23:46,356 DEBUG   HandlerThread:26728 [system_info.py:_save_pip():67] Saving pip packages done
2023-03-17 16:23:46,357 INFO    HandlerThread:26728 [system_monitor.py:probe():219] Finished publishing system info
2023-03-17 16:23:46,374 DEBUG   SenderThread:26728 [sender.py:send():336] send: files
2023-03-17 16:23:46,374 INFO    SenderThread:26728 [sender.py:_save_file():1332] saving file wandb-metadata.json with policy now
2023-03-17 16:23:46,392 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: stop_status
2023-03-17 16:23:46,393 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: stop_status
2023-03-17 16:23:46,675 DEBUG   SenderThread:26728 [sender.py:send():336] send: telemetry
2023-03-17 16:23:46,675 DEBUG   SenderThread:26728 [sender.py:send():336] send: telemetry
2023-03-17 16:23:46,675 DEBUG   SenderThread:26728 [sender.py:send():336] send: run
2023-03-17 16:23:46,908 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_modified():295] file/dir modified: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\config.yaml
2023-03-17 16:23:46,910 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_created():278] file/dir created: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\wandb-summary.json
2023-03-17 16:23:46,910 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_created():278] file/dir created: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\wandb-metadata.json
2023-03-17 16:23:46,911 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_created():278] file/dir created: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\requirements.txt
2023-03-17 16:23:46,998 INFO    SenderThread:26728 [sender.py:send_run():955] updated run: dytlg310
2023-03-17 16:23:47,297 INFO    wandb-upload_0:26728 [upload_job.py:push():138] Uploaded file C:\Users\HP\AppData\Local\Temp\tmppprnjkxfwandb\rvkjkwwv-wandb-metadata.json
2023-03-17 16:23:48,284 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:48,284 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:48,450 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:48,452 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:50,022 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:23:50,469 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:50,470 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:50,657 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:50,657 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:52,671 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:52,671 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:52,863 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:52,864 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:54,878 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:54,878 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:55,059 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:23:55,063 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:55,064 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:57,072 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:57,073 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:57,248 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:57,248 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:59,263 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:59,263 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:23:59,451 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:23:59,452 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:00,082 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:24:01,410 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: stop_status
2023-03-17 16:24:01,411 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: stop_status
2023-03-17 16:24:01,461 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:01,461 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:01,648 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:01,648 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:03,664 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:03,664 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:03,854 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:03,855 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:05,770 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:24:05,860 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:05,860 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:06,040 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:06,040 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:08,054 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:08,055 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:08,242 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:08,242 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:08,943 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: partial_history
2023-03-17 16:24:08,943 DEBUG   SenderThread:26728 [sender.py:send():336] send: history
2023-03-17 16:24:08,943 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: summary_record
2023-03-17 16:24:08,949 INFO    SenderThread:26728 [sender.py:_save_file():1332] saving file wandb-summary.json with policy end
2023-03-17 16:24:09,084 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_modified():295] file/dir modified: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\wandb-summary.json
2023-03-17 16:24:10,252 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:10,252 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:10,441 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:10,441 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:10,973 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:24:12,462 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:12,462 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:12,647 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:12,648 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:14,656 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:14,657 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:14,825 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:14,826 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:16,025 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:24:16,419 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: stop_status
2023-03-17 16:24:16,419 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: stop_status
2023-03-17 16:24:16,844 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:16,844 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:17,031 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:17,033 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:17,150 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_modified():295] file/dir modified: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\config.yaml
2023-03-17 16:24:19,270 ERROR   gpu       :26728 [interfaces.py:aggregate():158] Failed to serialize metric: division by zero
2023-03-17 16:24:19,278 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:19,279 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:21,287 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:21,287 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:21,474 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:21,474 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:21,744 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:24:23,497 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:23,497 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:23,684 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:23,685 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:25,701 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:25,702 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:25,888 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:25,889 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:26,768 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:24:27,906 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:27,906 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:28,097 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:28,097 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:30,107 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:30,108 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:30,273 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:30,273 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:30,866 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: partial_history
2023-03-17 16:24:30,866 DEBUG   SenderThread:26728 [sender.py:send():336] send: history
2023-03-17 16:24:30,866 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: summary_record
2023-03-17 16:24:30,866 INFO    SenderThread:26728 [sender.py:_save_file():1332] saving file wandb-summary.json with policy end
2023-03-17 16:24:31,258 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_modified():295] file/dir modified: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\wandb-summary.json
2023-03-17 16:24:31,433 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: stop_status
2023-03-17 16:24:31,434 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: stop_status
2023-03-17 16:24:32,278 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:32,278 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:32,470 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:32,470 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:32,749 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:24:34,490 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:34,491 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:34,667 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:34,667 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:36,678 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:36,678 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:36,852 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:36,852 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:37,780 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:24:38,862 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:38,862 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:39,032 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:39,032 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:41,043 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:41,043 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:41,221 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:41,221 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:42,810 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:24:43,228 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:43,228 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:43,403 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:43,404 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:45,409 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:45,409 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:45,592 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:45,592 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:46,260 DEBUG   SystemMonitor:26728 [system_monitor.py:_start():161] Starting system metrics aggregation loop
2023-03-17 16:24:46,261 DEBUG   SenderThread:26728 [sender.py:send():336] send: stats
2023-03-17 16:24:46,443 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: stop_status
2023-03-17 16:24:46,443 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: stop_status
2023-03-17 16:24:47,612 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:47,612 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:47,798 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:47,798 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:48,776 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:24:49,811 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:49,811 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:49,997 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:49,997 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:52,113 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: partial_history
2023-03-17 16:24:52,115 DEBUG   SenderThread:26728 [sender.py:send():336] send: history
2023-03-17 16:24:52,115 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: summary_record
2023-03-17 16:24:52,115 INFO    SenderThread:26728 [sender.py:_save_file():1332] saving file wandb-summary.json with policy end
2023-03-17 16:24:52,231 ERROR   gpu       :26728 [interfaces.py:aggregate():158] Failed to serialize metric: division by zero
2023-03-17 16:24:52,239 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:52,239 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:52,381 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_modified():295] file/dir modified: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\wandb-summary.json
2023-03-17 16:24:54,141 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:24:54,258 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:54,258 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:54,445 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:54,445 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:56,461 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:56,461 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:56,650 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:56,651 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:58,659 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:58,659 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:24:58,846 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:58,846 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:24:59,187 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:25:00,853 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:00,853 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:01,034 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:01,034 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:01,464 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: stop_status
2023-03-17 16:25:01,464 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: stop_status
2023-03-17 16:25:03,042 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:03,042 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:03,230 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:03,238 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:04,787 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:25:05,255 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:05,255 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:05,442 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:05,445 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:07,448 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:07,448 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:07,628 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:07,628 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:09,667 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:09,667 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:09,817 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:25:09,837 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:09,837 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:11,845 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:11,845 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:12,024 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:12,024 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:13,608 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: partial_history
2023-03-17 16:25:13,609 DEBUG   SenderThread:26728 [sender.py:send():336] send: history
2023-03-17 16:25:13,609 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: summary_record
2023-03-17 16:25:13,614 INFO    SenderThread:26728 [sender.py:_save_file():1332] saving file wandb-summary.json with policy end
2023-03-17 16:25:13,760 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_modified():295] file/dir modified: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\wandb-summary.json
2023-03-17 16:25:14,037 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:14,037 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:14,216 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:14,216 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:15,635 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:25:16,228 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:16,228 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:16,267 DEBUG   SenderThread:26728 [sender.py:send():336] send: stats
2023-03-17 16:25:16,410 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:16,411 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:16,468 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: stop_status
2023-03-17 16:25:16,469 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: stop_status
2023-03-17 16:25:18,430 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:18,430 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:18,618 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:18,618 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:20,633 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:20,634 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:20,816 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:20,817 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:21,042 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:25:22,826 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:22,826 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:23,008 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:23,008 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:25,218 ERROR   gpu       :26728 [interfaces.py:aggregate():158] Failed to serialize metric: division by zero
2023-03-17 16:25:25,227 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:25,227 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:26,079 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:25:27,239 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:27,239 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:27,424 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:27,425 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:29,431 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:29,431 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:29,602 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:29,603 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:31,119 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:25:31,481 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: stop_status
2023-03-17 16:25:31,482 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: stop_status
2023-03-17 16:25:31,614 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:31,614 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:31,797 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:31,798 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:33,809 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:33,809 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:34,012 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:34,014 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:36,028 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:36,028 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Unknown Error
2023-03-17 16:25:36,212 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:36,212 ERROR   gpu       :26728 [interfaces.py:monitor():141] Failed to sample metric: Not Supported
2023-03-17 16:25:36,876 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:25:37,015 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: partial_history
2023-03-17 16:25:37,015 DEBUG   SenderThread:26728 [sender.py:send():336] send: history
2023-03-17 16:25:37,015 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: summary_record
2023-03-17 16:25:37,015 INFO    SenderThread:26728 [sender.py:_save_file():1332] saving file wandb-summary.json with policy end
2023-03-17 16:25:37,157 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_modified():295] file/dir modified: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\wandb-summary.json
2023-03-17 16:25:37,632 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: partial_history
2023-03-17 16:25:37,633 DEBUG   SenderThread:26728 [sender.py:send():336] send: files
2023-03-17 16:25:37,635 INFO    SenderThread:26728 [sender.py:_save_file():1332] saving file media\images\Confusion Matrix_5_5a936bec8508e86ae7f5.png with policy now
2023-03-17 16:25:37,636 DEBUG   SenderThread:26728 [sender.py:send():336] send: history
2023-03-17 16:25:37,636 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: summary_record
2023-03-17 16:25:37,638 INFO    SenderThread:26728 [sender.py:_save_file():1332] saving file wandb-summary.json with policy end
2023-03-17 16:25:37,639 DEBUG   SenderThread:26728 [sender.py:send():336] send: telemetry
2023-03-17 16:25:37,639 DEBUG   SenderThread:26728 [sender.py:send():336] send: exit
2023-03-17 16:25:37,639 INFO    SenderThread:26728 [sender.py:send_exit():559] handling exit code: 0
2023-03-17 16:25:37,639 INFO    SenderThread:26728 [sender.py:send_exit():561] handling runtime: 111
2023-03-17 16:25:37,639 INFO    SenderThread:26728 [sender.py:_save_file():1332] saving file wandb-summary.json with policy end
2023-03-17 16:25:37,639 INFO    SenderThread:26728 [sender.py:send_exit():567] send defer
2023-03-17 16:25:37,639 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:37,639 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 0
2023-03-17 16:25:37,639 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:37,639 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 0
2023-03-17 16:25:37,639 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 1
2023-03-17 16:25:37,639 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:37,639 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 1
2023-03-17 16:25:37,639 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:37,644 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 1
2023-03-17 16:25:37,644 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 2
2023-03-17 16:25:37,644 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:37,644 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 2
2023-03-17 16:25:37,644 INFO    HandlerThread:26728 [system_monitor.py:finish():193] Stopping system monitor
2023-03-17 16:25:37,646 DEBUG   SystemMonitor:26728 [system_monitor.py:_start():168] Finished system metrics aggregation loop
2023-03-17 16:25:37,646 DEBUG   SystemMonitor:26728 [system_monitor.py:_start():172] Publishing last batch of metrics
2023-03-17 16:25:37,658 INFO    HandlerThread:26728 [interfaces.py:finish():199] Joined cpu monitor
2023-03-17 16:25:37,658 INFO    HandlerThread:26728 [interfaces.py:finish():199] Joined disk monitor
2023-03-17 16:25:37,864 ERROR   gpu       :26728 [interfaces.py:aggregate():158] Failed to serialize metric: division by zero
2023-03-17 16:25:37,864 INFO    HandlerThread:26728 [interfaces.py:finish():199] Joined gpu monitor
2023-03-17 16:25:37,864 INFO    HandlerThread:26728 [interfaces.py:finish():199] Joined memory monitor
2023-03-17 16:25:37,865 INFO    HandlerThread:26728 [interfaces.py:finish():199] Joined network monitor
2023-03-17 16:25:37,865 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:37,865 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 2
2023-03-17 16:25:37,865 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 3
2023-03-17 16:25:37,865 DEBUG   SenderThread:26728 [sender.py:send():336] send: stats
2023-03-17 16:25:37,865 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:37,865 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 3
2023-03-17 16:25:37,865 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:37,865 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 3
2023-03-17 16:25:37,865 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 4
2023-03-17 16:25:37,865 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:37,865 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 4
2023-03-17 16:25:37,865 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:37,865 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 4
2023-03-17 16:25:37,865 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 5
2023-03-17 16:25:37,865 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:37,865 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 5
2023-03-17 16:25:37,865 DEBUG   SenderThread:26728 [sender.py:send():336] send: summary
2023-03-17 16:25:37,865 INFO    SenderThread:26728 [sender.py:_save_file():1332] saving file wandb-summary.json with policy end
2023-03-17 16:25:37,865 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:37,865 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 5
2023-03-17 16:25:37,865 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 6
2023-03-17 16:25:37,865 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:37,865 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 6
2023-03-17 16:25:37,865 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:37,865 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 6
2023-03-17 16:25:37,874 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: status_report
2023-03-17 16:25:38,158 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_modified():295] file/dir modified: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\wandb-summary.json
2023-03-17 16:25:38,158 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_created():278] file/dir created: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\media\images\Confusion Matrix_5_5a936bec8508e86ae7f5.png
2023-03-17 16:25:38,159 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_created():278] file/dir created: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\media\images
2023-03-17 16:25:38,160 INFO    Thread-17 :26728 [dir_watcher.py:_on_file_created():278] file/dir created: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\media
2023-03-17 16:25:38,197 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 7
2023-03-17 16:25:38,198 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:38,198 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 7
2023-03-17 16:25:38,198 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:38,199 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 7
2023-03-17 16:25:38,199 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 8
2023-03-17 16:25:38,199 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:38,199 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 8
2023-03-17 16:25:38,200 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:38,200 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 8
2023-03-17 16:25:38,200 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 9
2023-03-17 16:25:38,201 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:38,201 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 9
2023-03-17 16:25:38,202 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:38,202 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 9
2023-03-17 16:25:38,202 INFO    SenderThread:26728 [dir_watcher.py:finish():365] shutting down directory watcher
2023-03-17 16:25:38,484 INFO    wandb-upload_0:26728 [upload_job.py:push():138] Uploaded file C:\Users\HP\AppData\Local\Temp\tmppprnjkxfwandb\9uzs8fix-media/images/Confusion Matrix_5_5a936bec8508e86ae7f5.png
2023-03-17 16:25:38,650 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: poll_exit
2023-03-17 16:25:39,165 INFO    SenderThread:26728 [dir_watcher.py:_on_file_modified():295] file/dir modified: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\config.yaml
2023-03-17 16:25:39,165 INFO    SenderThread:26728 [dir_watcher.py:finish():395] scan: d:\CS6901\wandb\run-20230317_162344-dytlg310\files
2023-03-17 16:25:39,165 INFO    SenderThread:26728 [dir_watcher.py:finish():409] scan save: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\config.yaml config.yaml
2023-03-17 16:25:39,166 INFO    SenderThread:26728 [dir_watcher.py:finish():409] scan save: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\requirements.txt requirements.txt
2023-03-17 16:25:39,169 INFO    SenderThread:26728 [dir_watcher.py:finish():409] scan save: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\wandb-metadata.json wandb-metadata.json
2023-03-17 16:25:39,169 INFO    SenderThread:26728 [dir_watcher.py:finish():409] scan save: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\wandb-summary.json wandb-summary.json
2023-03-17 16:25:39,170 INFO    SenderThread:26728 [dir_watcher.py:finish():409] scan save: d:\CS6901\wandb\run-20230317_162344-dytlg310\files\media\images\Confusion Matrix_5_5a936bec8508e86ae7f5.png media\images\Confusion Matrix_5_5a936bec8508e86ae7f5.png
2023-03-17 16:25:39,170 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 10
2023-03-17 16:25:39,170 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: poll_exit
2023-03-17 16:25:39,172 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:39,173 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 10
2023-03-17 16:25:39,173 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:39,173 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 10
2023-03-17 16:25:39,173 INFO    SenderThread:26728 [file_pusher.py:finish():164] shutting down file pusher
2023-03-17 16:25:39,658 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: poll_exit
2023-03-17 16:25:39,659 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: poll_exit
2023-03-17 16:25:39,929 INFO    wandb-upload_2:26728 [upload_job.py:push():138] Uploaded file d:\CS6901\wandb\run-20230317_162344-dytlg310\files\wandb-summary.json
2023-03-17 16:25:40,048 INFO    wandb-upload_0:26728 [upload_job.py:push():138] Uploaded file d:\CS6901\wandb\run-20230317_162344-dytlg310\files\config.yaml
2023-03-17 16:25:40,670 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: poll_exit
2023-03-17 16:25:40,671 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: poll_exit
2023-03-17 16:25:40,708 INFO    wandb-upload_1:26728 [upload_job.py:push():138] Uploaded file d:\CS6901\wandb\run-20230317_162344-dytlg310\files\requirements.txt
2023-03-17 16:25:40,921 INFO    Thread-16 :26728 [sender.py:transition_state():587] send defer: 11
2023-03-17 16:25:40,921 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:40,921 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 11
2023-03-17 16:25:40,921 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:40,921 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 11
2023-03-17 16:25:40,921 INFO    SenderThread:26728 [file_pusher.py:join():169] waiting for file pusher
2023-03-17 16:25:40,922 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 12
2023-03-17 16:25:40,922 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:40,922 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 12
2023-03-17 16:25:40,922 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:40,922 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 12
2023-03-17 16:25:41,684 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: poll_exit
2023-03-17 16:25:41,812 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 13
2023-03-17 16:25:41,812 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: poll_exit
2023-03-17 16:25:41,813 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:41,813 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 13
2023-03-17 16:25:41,813 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:41,813 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 13
2023-03-17 16:25:41,813 INFO    SenderThread:26728 [sender.py:transition_state():587] send defer: 14
2023-03-17 16:25:41,813 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: defer
2023-03-17 16:25:41,813 INFO    HandlerThread:26728 [handler.py:handle_request_defer():170] handle defer: 14
2023-03-17 16:25:41,814 DEBUG   SenderThread:26728 [sender.py:send():336] send: final
2023-03-17 16:25:41,814 DEBUG   SenderThread:26728 [sender.py:send():336] send: footer
2023-03-17 16:25:41,814 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: defer
2023-03-17 16:25:41,814 INFO    SenderThread:26728 [sender.py:send_request_defer():583] handle sender defer: 14
2023-03-17 16:25:41,815 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: poll_exit
2023-03-17 16:25:41,816 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: server_info
2023-03-17 16:25:41,816 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: get_summary
2023-03-17 16:25:41,816 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: sampled_history
2023-03-17 16:25:41,817 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: poll_exit
2023-03-17 16:25:41,817 DEBUG   SenderThread:26728 [sender.py:send_request():363] send_request: server_info
2023-03-17 16:25:42,440 DEBUG   HandlerThread:26728 [handler.py:handle_request():144] handle_request: shutdown
2023-03-17 16:25:42,440 INFO    HandlerThread:26728 [handler.py:finish():842] shutting down handler
2023-03-17 16:25:42,825 INFO    WriterThread:26728 [datastore.py:close():298] close: d:\CS6901\wandb\run-20230317_162344-dytlg310\run-dytlg310.wandb
2023-03-17 16:25:43,451 INFO    SenderThread:26728 [sender.py:finish():1504] shutting down sender
2023-03-17 16:25:43,451 INFO    SenderThread:26728 [file_pusher.py:finish():164] shutting down file pusher
2023-03-17 16:25:43,451 INFO    SenderThread:26728 [file_pusher.py:join():169] waiting for file pusher
